be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.de.resx

```diff
@@ -144,6 +144,9 @@
   <data name="EmailIsConfirmedText" xml:space="preserve">
     <value>Konto bestätigt</value>
   </data>
+  <data name="EmailMarketingApprovalLabel" xml:space="preserve">
+    <value>Ich erteile meine Zustimmung zum Erhalt von Handelsinformationen von ELTAP auf elektronischem Wege an die von mir angegebene E-Mail-Adresse</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP